### PR TITLE
Allow Chotify  to play sounds

### PR DESCRIPTION
**Pull request by TurBoss**

### CODE REVIEW

File: libs/chotify/chotify/chotify.lua [hunk 1]
```diff
@@ -39,14 +39,19 @@ function Chotify:Post(obj)
     local body = obj.body or ""
     local icon = obj.icon or ""
     local time = obj.time or 5
+    local sound = obj.sound or nil
+    
+    if sound then
+        Spring.PlaySoundFile("sounds/" .. sound, 1.0)
```
Comment (gajop) [inline libs/chotify/chotify/chotify.lua]: No need to enforce the path here. It's not difficult writing the full path.

**Comment by TurBoss** [comment]
done

File: libs/chotify/chotify/chotify.lua [hunk 2]
```diff
@@ -39,14 +39,19 @@ function Chotify:Post(obj)
     local body = obj.body or ""
     local icon = obj.icon or ""
     local time = obj.time or 5
+    local sound = obj.sound or nil
```
Comment (gajop) [inline libs/chotify/chotify/chotify.lua]: "or nil" makes no sense here. The point of "or X" is to provide default values, which sound doesn't need (yet).

Comment (TurBoss) [inline libs/chotify/chotify/chotify.lua]: should be 
````
  local sound = obj.sound 
```
?

Comment (gajop) [inline libs/chotify/chotify/chotify.lua]: Yes, or you can directly use obj.sound, it doesn't matter.

**Pull request merged**